js/bootstrap-datepicker.js

```diff
@@ -262,6 +262,7 @@
 				});
 				o.orientation.y = _plc[0] || 'auto';
 			}
+			o.container = o.container ? o.container : 'body';
```

Comment:
I've merged this PR and fixed the note myself, this feature will be included in the 1.4.0 release!